airbyte-integrations/connectors/destination-azure-blob-storage/src/main/resources/spec.json

```diff
@@ -35,13 +35,20 @@
         "examples": ["airbyte5storage"]
       },
       "azure_blob_storage_account_key": {
+        "title": "Azure Blob Storage account key",
         "description": "The Azure blob storage account key.",
         "airbyte_secret": true,
         "type": "string",
         "examples": [
           "Z8ZkZpteggFx394vm+PJHnGTvdRncaYS+JhLKdj789YNmD+iyGTnG+PV+POiuYNhBg/ACS+LKjd%4FG3FHGN12Nd=="
         ]
       },
+      "azure_blob_storage_output_buffer_size": {
```

Comment:
Should there be a default key and value here?

Reply:
It does default to 100MB, however I went back to docs to see if I could find something on defaults, but what I'm finding is pretty limited. I don't see a way to specify a default value for a property in the connectionSpecification. If you can point me to information about it I'll add it in though. Would be nice to have a range too, since you can technically put larger than a 32bit signed int in the field and it currently rolls over. Logic in the code will force it to 100MB if its <= 0 though.

Reply:
Disregard, dug through other integrations and found an example, would be nice if it were on the doc page though https://docs.airbyte.com/connector-development/connector-specification-reference